src/components/MovieApp/MovieApp.tsx

```diff
@@ -1,9 +1,126 @@
+    rating: 5,
+  },
+  {
+    id: 234,
+    name: 'Kill Bill: Vol 2',
+    preview: 'https://picsum.photos/id/1028/600/700',
+    genre: 'drama',
+    year: 2018,
+    duration: 153,
+    description: 'Pulp Fiction is a 1994 American crime film written and directed by Quentin Tarantino; it is based on a story by Tarantino and Roger Avary.[4] Starring John Travolta, Samuel L. Jackson, Bruce Willis, Tim Roth, Ving Rhames, and Uma Thurman, it tells several stories of criminal Los Angeles. The film\'s title refers to the pulp magazines and hardboiled crime novels popular during the mid-20th century, known for their graphic violence and punchy dialogue.',
+    rating: 4.3,
+  },
+  {
+    id: 10,
+    name: 'Kill Bill: Vol 4',
+    preview: 'https://picsum.photos/id/1028/600/700',
+    genre: 'drama',
+    year: 2017,
+    duration: 153,
+    description: 'Pulp Fiction is a 1994 American crime film written and directed by Quentin Tarantino; it is based on a story by Tarantino and Roger Avary.[4] Starring John Travolta, Samuel L. Jackson, Bruce Willis, Tim Roth, Ving Rhames, and Uma Thurman, it tells several stories of criminal Los Angeles. The film\'s title refers to the pulp magazines and hardboiled crime novels popular during the mid-20th century, known for their graphic violence and punchy dialogue.',
+    rating: 5,
+  },
+  {
+    id: 167,
+    name: 'Kill Bill: Vol 3',
+    preview: 'https://picsum.photos/id/1028/600/700',
+    genre: 'drama',
+    year: 2005,
+    duration: 153,
+    description: 'Pulp Fiction is a 1994 American crime film written and directed by Quentin Tarantino; it is based on a story by Tarantino and Roger Avary.[4] Starring John Travolta, Samuel L. Jackson, Bruce Willis, Tim Roth, Ving Rhames, and Uma Thurman, it tells several stories of criminal Los Angeles. The film\'s title refers to the pulp magazines and hardboiled crime novels popular during the mid-20th century, known for their graphic violence and punchy dialogue.',
+    rating: 4.8,
+  },
+];
 
-export class MovieApp extends React.Component {
-  render() {
-    return <h1 className={css.container}>Movies App</h1>
-  }
+export function MovieApp () {
+  const [searchType, setSearchType] = useState('title');
+  const [sort, setSort] = useState('release date');
+  const [activeCard, setActiveCard] = useState(null);
+  const [films, setFilms] = useState([]);
+
+  const sortItems = (val: string) => {
+    const sort = (val === 'rating') ? val : 'year';
+    setSort(val);
+
+    films.sort((first, next) => {
```

Comment:
```suggestion
    films.slice().sort((first, next) => {
```

ради иммутабельности

Reply:
ну и `setFilms` после этого